codalab/server/json_api_plugin.py

```diff
@@ -0,0 +1,31 @@
+from bottle import (
+  abort,
+  httplib,
+  redirect,
+  request,
+  url,
+)
+from marshmallow import ValidationError
+
+from codalab.lib.server_util import query_get_bool
+
+
+class JsonApiPlugin(object):
```

Comment:
Seems like ValidationErrors are thrown only by JSONAPI methods. Why not always run this plugin?